ov/firmware_drivers.go

```diff
@@ -145,6 +147,48 @@ func (c *OVClient) GetFirmwareBaselineById(id string) (FirmwareDrivers, error) {
 	return firmwareId, nil
 }
 
+func (c *OVClient) GetFirmwareBaselineByNameandVersion(name string) (FirmwareDrivers, error) {
+	var fwname, version string
+	fwNameVersion := strings.SplitAfter(name, ",")
+	if len(fwNameVersion) < 1 {
+		return FirmwareDrivers{}, errors.New("firmware name not provided")
+	}
+	if len(fwNameVersion) == 2 {
+		fwname, version = fwNameVersion[0], fwNameVersion[1]
+	} else {
+		fwname = fwNameVersion[0]
+	}
+
+	firmwareList, err := c.GetFirmwareBaselineList("", "", "")
+
+	if firmwareList.Total > 0 {
+
+		for i := range firmwareList.Members {
+			if version != "" {
+				if firmwareList.Members[i].Name != fwname && firmwareList.Members[i].Version != version {
```

Comment:
this is case sensitive, user has to provide exact name....anyway fine for now..let see if we get anything on this...